README.md

```diff
@@ -94,3 +94,15 @@ This API will return Bridge Identifier by which you can use while sending SMS
 		}]
 
 Note: While sending SMS, don't include country code as part of each message. This country code will be prefixed by taking value from provider details. You need configure host-address: [your public ip on which message-gateway is running] in application.yml
+
+#### To add Telerivet Java Client : Run the following commands 
+
+    git clone https://github.com/Telerivet/telerivet-java-client
+    mvn install
+
```

Comment:
@ankita10r : See JFrog URL posted in Group.

Reply:
ok

Reply:
Resolved with PR https://github.com/openMF/message-gateway/pull/28